Gemfile.lock

```diff
@@ -55,20 +55,20 @@ GEM
       ransack (~> 2.1, >= 2.1.1)
       sassc-rails (~> 2.1)
       sprockets (>= 3.0, < 4.1)
-    activejob (6.0.2.1)
-      activesupport (= 6.0.2.1)
+    activejob (6.0.2.2)
```

Comment:
### [activejob](https://rubyonrails.org)



[_[change-log](https://github.com/rails/rails/blob/v6.0.2.2/activejob/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.2.2/activejob)_]





<details>
<summary>Commits</summary>

A change of **3** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.2.1...v6.0.2.2).

These are the individual commits:
- (1251d88) [Fix possible XSS vector in JS escape helper](https://github.com/rails/rails/commit/1251d8817264744163fb12a3dba05ce61be5371b)
- (fa413f2) [update version](https://github.com/rails/rails/commit/fa413f2e8411cb22f6d76b119a79a25aab1e935c)
- (157920a) [Preparing for 6.0.2.2 release](https://github.com/rails/rails/commit/157920aead96865e3135f496c09ace607d5620dc)


</details>